### PR TITLE
Vorbis

### PR DESCRIPTION
**Pull request by vlj**

### CODE REVIEW

File: ports/libvorbis/configure.ac [hunk 1]
```diff
@@ -0,0 +1,304 @@
+dnl Process this file with autoconf to produce a configure script
```
Comment (Orvid) [inline ports/libvorbis/configure.ac]: Why the configure scripts? They don't actually get used do they?

**Comment by ras0219-msft** [comment]
I looked at their most recent commits[1][2], and it looks like the two or three dozen commits over the last few years have been almost entirely for compatibility and CMake. I think given the maturity of the project, it makes sense to pull the latest snapshot which already has all these files present: ex. https://git.xiph.org/?p=ogg.git;a=snapshot;h=cab46b19847eb1d38aeda73ab4708716e5776366;sf=tgz. 

Would you mind rebasing to use the snapshots directly instead of checking the backported files in?

[1] https://git.xiph.org/?p=ogg.git;a=shortlog
[2] https://git.xiph.org/?p=vorbis.git;a=shortlog

**Comment by vlj** [comment]
The SHA from the downloaded file changes every time. Is there a way to clone git repo/revision ?

**Comment by vlj** [comment]
nvm I found cpprestsdk is using git too.

**Comment by ras0219-msft** [comment]
The pages I added above have a "snapshot" link by each commit that allows you to download the archives directly (no git clone required).

**Comment by vlj** [comment]
I modified the portfiles.cmake so that they use git + revision instead of bundling CMakeLists.txt from the git repo.

Unfortunatly downloading archive directly didn't work, for some reason the SHA512 didn't match when downloading several time the same snapshot from git.xiph.org and I couldn't bypass the SHA check.

**Comment by ras0219-msft** [comment]
Ah, I didn't read your previous comments closely enough. Yeah, cloning from git seems to be the only option then. I'll try out these builds locally and if all goes well merge them.

**Comment by ras0219-msft** [comment]
Thanks! I added calls to `vcpkg_copy_pdbs()` to copy the built PDBs alongside the DLLs.

**Pull request merged**